docs/tutorials/05_torch_connector.ipynb

```diff
@@ -717,10 +692,7 @@
     "plt.plot(X, y, 'bo')\n",
     "\n",
     "# Plot fitted line\n",
-    "y_ = []\n",
-    "for x in np.linspace(lb, ub):\n",
-    "    output = model3(Tensor([x]))\n",
-    "    y_ += [output.detach().numpy()[0]]\n",
+    "y_ = model3(Tensor(np.linspace(lb, ub)).unsqueeze(-1)).detach().numpy().squeeze()\n",
```

Comment:
Over-complicated line for the tutorial, requires explanations. I suggest at least split it into two, e.g.
```python
input_data = Tensor(np.linspace(lb, ub)).unsqueeze(-1)
y_ = model3(input_data).detach().numpy().squeeze()
```
and add comments.